app/models/location.rb

```diff
@@ -0,0 +1,18 @@
+class Location < ActiveRecord::Base
+  belongs_to :sensor
+
+  validates :latitude, :longitude, :sensor, presence: true
+  validates :latitude, numericality: { greater_than_or_equal_to: -90, less_than_or_equal_to: 90 }
+  validates :longitude, numericality: { greater_than_or_equal_to: -180, less_than_or_equal_to: 180 }
+
+  before_create :set_registration_time
+
+  private
+
+  def set_registration_time
+    self.registration_time = Time.now
```

Comment:
I prefer it this way. As Locations are not as numerous as Readings, so saving space is not required, I prefer to have a field that is "semantically" correct and serves its on, single, purpose.